### PR TITLE
Update route for chapter 31 and turn on production build.

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request turns on the production build for Chapter 31 and also updates the root url for the application itself per feedback from IA. 

## Testing done
- local

## Acceptance criteria
- [ ] e2e tests pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by Mottie — APPROVED** [review]
ghe-emoji

**Review by timwright12 — APPROVED** [review]

File: src/applications/static-pages/vre-chapter31/Chapter31CTA.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Chapter31CTA = props => {
     content = (
       <>
         <a
-          href="/careers-employment/vocational-rehabilitation/apply"
+          href="/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900"
```
Comment (Cgfor3) [inline src/applications/static-pages/vre-chapter31/Chapter31CTA.js:32]: Is it worth replacing this string with the constant defined in `src/applications/vre/28-1900/constants.js`

**Review by jsev-io — APPROVED** [review]

File: src/applications/static-pages/vre-chapter31/Chapter31CTA.js (hunk 1 above)
Comment (ncksllvn) [reply src/applications/static-pages/vre-chapter31/Chapter31CTA.js:32]: I agree @Cgfor3 but there is also some ambiguity on the concept of apps importing code from one another. We are kind of cautioned about doing that.

**Review by ncksllvn — APPROVED** [review]

File: src/applications/static-pages/vre-chapter31/Chapter31CTA.js (hunk 1 above)
Comment (micahchiang) [reply src/applications/static-pages/vre-chapter31/Chapter31CTA.js:32]: It's definitely not a bad idea, but given the length of these CI/CD runs, and my team's desire to have this in the daily deploy, I'll plan on implementing this in a future PR.

**Pull request merged**